ci-operator/config/openshift-priv/installer/openshift-priv-installer-main.yaml

```diff
@@ -553,9 +553,9 @@ tests:
     post:
     - chain: ipi-azure-post
     pre:
-    - ref: azure-provision-service-principal-minimal-permission
     - ref: ipi-conf
     - ref: ipi-conf-azure-default
+    - chain: azure-provision-service-principal-minimal-permission
```

Comment:
Do we really want to change `openshift-priv` ones ?

Reply:
Are those openshift-priv jobs automatically synced up by some way? I searched them when updating this chain.
I'm not sure when this job will be triggered. Once triggered, the change is needed here.

Reply:
Yeah, we updated the `openshift-priv` ones on purpose because of mini-perm change.

Reply:
AFAIK, those `openshift-priv` are used by ART team.  Generally we don't touch them as there are auto sync mechanism. 
If you are sure this is exactly needed, I can approve it. Just let me know.

Reply:
got it, the change is required regardless in which way to update them.
Thanks for approving this.

Reply:
@liangxia I removed the change in `openshift-priv` job config, PTAL.

Reply:
seems that the change must be done manually in this PR for `openshift-priv` jobs, otherwise job [pull-ci-openshift-release-master-ci-operator-config](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/61374/pull-ci-openshift-release-master-ci-operator-config/1894257025961955328) failed, because some envs are removed on purpose in this PR.

So I changed back for `openshift-priv` jobs.